src/macros/for_others.rs

```diff
@@ -110,6 +110,21 @@ macro_rules! implement_for_owned {
         }
     };
 
+    // [`From`] trait
+    ($source_type:ident, $type:ident, From) => {
+        #[doc(hidden)]
+        impl From<$source_type> for $type {
+            paste::paste! {
+                #[doc = "Documentation can be found at [`" $type "::evaluate`]."]
+            fn from(
+                value: $source_type
+            ) -> Self {
```

Comment:
```suggestion
            fn from(value: $source_type) -> Self {
```
Please, run the formatter over this again. But this part / the braces don't look like they were aligned correctly.